pyanaconda/modules/payload/live/initialization.py

```diff
@@ -156,6 +156,55 @@ def run(self):
         return size
 
 
+class DownloadProgress(object):
```

Comment:
Shouldn't this and the class using this be part of the `installation.py` file instead?

Reply:
I'll leave this to you or follow-up PRs. I think pre and especially post install (because pre for LiveImage is basically setup done in pre stage because it needs storage being created) could go into installation.py